### ISSUE TITLE
Upgrade to v2.10 breaks working program

### ISSUE THREAD

**Issue by anvabr** [hashgraph/hedera-sdk-js#1000]
### Description

We upgraded SDK from v2.6 to 2.10 and it broke our app, we think the last working version is v2.7. Investigating the issue it appears that `requestTimeout` was added into execute method of all transactions. There seem to be a problem in the method may be to do with comparison, as with any `requestTimeout > 0 `it will fail with an error. In our case the error is:
```
(new TopicMessageSubmitTransaction({
        topicId: topicId,
        message: message,
})).execute(client);
```
if message is >1KB and it tries to split it in two.

### Steps to reproduce

See description

### Hedera network

testnet

### Version

v2.10

**Comment by danielakhterov** [comment]
I think I discovered this issue later on as well, and it _might_ be fixed by 2.11.0-beta.1 Could you give that a shot to confirm?

**Comment by Stepan-Kirjakov** [comment]
Yes, this seems to be fixed in 2.11.0-beta.1

**Comment by danielakhterov** [comment]
Awesome! I'll close this issue now since we're releasing 2.11.0 Monday.
